src/nbl/video/CVulkanLogicalDevice.h

```diff
@@ -860,15 +860,15 @@ class CVulkanLogicalDevice final : public ILogicalDevice
         vk_createInfo.addressModeV = getVkAddressModeFromTexClamp(static_cast<asset::ISampler::E_TEXTURE_CLAMP>(_params.TextureWrapV));
         vk_createInfo.addressModeW = getVkAddressModeFromTexClamp(static_cast<asset::ISampler::E_TEXTURE_CLAMP>(_params.TextureWrapW));
         vk_createInfo.mipLodBias = _params.LodBias;
-        vk_createInfo.anisotropyEnable = _params.AnisotropicFilter; // Todo(achal): Verify
-        vk_createInfo.maxAnisotropy = std::log2(_params.AnisotropicFilter); // Todo(achal): Verify
+        vk_createInfo.maxAnisotropy = std::exp2(_params.AnisotropicFilter); // Todo(achal): Verify
+        vk_createInfo.anisotropyEnable = VK_FALSE; // vk_createInfo.maxAnisotropy < 1.f ? VK_FALSE : VK_TRUE; // Todo(achal): Verify. Also this needs to take into account if the anistropic sampling feature is enabled
```

Comment:
you mean the driver feature (as in extension)?

Reply:
No, not an extension, a __feature__.
`VkPhysicalDeviceFeatures::samplerAnisotropy`

Reply:
is it exposed in our `SFeatures` yet?

Reply:
Not yet

Reply:
@Erfan-Ahmadi expose anisotropy support in `uint8_t SFeatures::maxAnisotropyLog2` (0 means unsupported / 1x)

Always enable if available on the Vulkan Logical device, seems stupid to ask user for an opinion.